### ISSUE TITLE
V2 - Getting TypeError when closing executor as STOP_LOSS

### ISSUE THREAD

**Issue by rapcmia** [hummingbot/hummingbot#7245]
### Describe the bug

When reviewing long term bot, found an error when closing executor as stop-loss. Tried to do a quick tests on another connector like kucoin and easily reproduced by setting up a config with tight STOP_LOSS spreads 
```
2024-10-07 12:58:02,931 - 70929 - hummingbot.strategy_v2.runnable_base - ERROR - unsupported operand type(s) for -: 'float' and 'NoneType'
Traceback (most recent call last):
  File "/Users/rapcomia/github/hummingbot/hummingbot/dev-hummingbot/hummingbot/strategy_v2/runnable_base.py", line 69, in control_loop
    await self.control_task()
  File "/Users/rapcomia/github/hummingbot/hummingbot/dev-hummingbot/hummingbot/strategy_v2/controllers/controller_base.py", line 170, in control_task
    executor_actions: List[ExecutorAction] = self.determine_executor_actions()
  File "/Users/rapcomia/github/hummingbot/hummingbot/dev-hummingbot/hummingbot/strategy_v2/controllers/market_making_controller_base.py", line 231, in determine_executor_actions
    actions.extend(self.create_actions_proposal())
  File "/Users/rapcomia/github/hummingbot/hummingbot/dev-hummingbot/hummingbot/strategy_v2/controllers/market_making_controller_base.py", line 240, in create_actions_proposal
    levels_to_execute = self.get_levels_to_execute()
  File "/Users/rapcomia/github/hummingbot/hummingbot/dev-hummingbot/hummingbot/strategy_v2/controllers/market_making_controller_base.py", line 252, in get_levels_to_execute
    working_levels = self.filter_executors(
  File "/Users/rapcomia/github/hummingbot/hummingbot/dev-hummingbot/hummingbot/strategy_v2/controllers/controller_base.py", line 182, in filter_executors
    return [executor for executor in executors if filter_func(executor)]
  File "/Users/rapcomia/github/hummingbot/hummingbot/dev-hummingbot/hummingbot/strategy_v2/controllers/controller_base.py", line 182, in <listcomp>
    return [executor for executor in executors if filter_func(executor)]
  File "/Users/rapcomia/github/hummingbot/hummingbot/dev-hummingbot/hummingbot/strategy_v2/controllers/market_making_controller_base.py", line 254, in <lambda>
    filter_func=lambda x: x.is_active or (x.close_type == CloseType.STOP_LOSS and self.market_data_provider.time() - x.close_timestamp < self.config.cooldown_time)
TypeError: unsupported operand type(s) for -: 'float' and 'NoneType'
```

Client screenshots 
![image](https://github.com/user-attachments/assets/bcb10287-8e0c-48f9-9184-0b8be1f3d258)
![image](https://github.com/user-attachments/assets/adf397be-bae6-4ce5-a197-8d531913234c)

### Steps to reproduce

1.  Setup a v2 strategy on any connector 
2. Setup tight spreads on stop loss to easily trigger
3. Observe behavior when closing executor 

### Release version

development 2.0.2

### Type of installation

Source

### Attach required files

[10072024_typeError-stoploss.zip](https://github.com/user-attachments/files/17273394/10072024_typeError-stoploss.zip)

**Comment by cardosofede** [comment]
@rapcmia this issue should be fixed in my #7244

**Comment by rapcmia** [comment]
Fix applied on Fede's PR
